.eslintrc.json

```diff
@@ -1,20 +1,36 @@
 {
-  "parserOptions": {
-    "ecmaVersion": 2017
-  },
   "extends": [
-    "eslint:recommended"
+    "eslint:recommended",
+    "plugin:security/recommended"
   ],
   "env": {
     "browser": true,
     "commonjs": true,
     "es6": true
   },
+  "parserOptions": {
+    "ecmaVersion": 2017
+  },
+  "plugins": [
+    "jsdoc",
+    "json",
+    "security"
+  ],
   "root": true,
   "rules": {
+    "jsdoc/check-types": "error",
+    "security/detect-non-literal-fs-filename": "off",
```

Comment:
This is generally a good rule to enable. I took a look at all the errors and nothing looked too risky. Alternatively, we could re-enable the rule and just do an `// eslint-disable-next-line security/detect-non-literal-fs-filename` or something for each usage that we've verified as safe.